deploy/chart/templates/0000_50_olm_00-namespace.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
     {{- if .Values.namespace_psa }}
     pod-security.kubernetes.io/enforce: {{ .Values.namespace_psa.enforceLevel }}
     pod-security.kubernetes.io/enforce-version: {{ .Values.namespace_psa.enforceVersion }}
+    pod-security.kubernetes.io/audit: restricted
+    pod-security.kubernetes.io/warn: restricted
```

Comment:
@joelanford I _think_ we want these here as well, thoughts?

Reply:
+1

Reply:
+1 we do want these here, but a nit is to have these also be populated from the chart value like the enforce values, to make things easier during downstream. eg if we want different values in downstream, we'll simply change the chart values.

It's a hygiene nit since we can always change the hardcoded value too, but that seems like confusing paper trail potentially.